code/datums/keybinding/artificial_intelligence.dm

```diff
@@ -0,0 +1,21 @@
+/datum/keybinding/artificial_intelligence
+	category = CATEGORY_AI
+	weight = WEIGHT_AI
+
+/datum/keybinding/artificial_intelligence/can_use(client/user)
+	return isAI(user.mob)
```

Comment:
This will cause issues because when in a shell the user is no longer typed to be the AI.
probably want to see if the user is a borg first and check for shell being true

Reply:
My original intent was that /artificial_intelligence hotkeys wouldn't be used while inside a shell (Right now for instance, there'd be no use to a deploy to shell hotkey while inside a shell). I'd have to see if it's compatible with can_use but I could probably implement a IsAIShell helper and put that as an OR in the return? I'd appreciate it if you could elaborate on these issues that you forsee, it might help me solve this is a neater way.

This could probably help address your last review as well, and get the disconnect hotkeys moved to this file in the process, but I'd have to check like !isAI(user) at the start of down(client/user) probably and I feel like that would be messy.